src/main/java/org/generationcp/middleware/api/germplasm/GermplasmService.java

```diff
@@ -109,13 +109,11 @@
 
 	PedigreeDTO getPedigree(Integer gid, String notation, Boolean includeSiblings);
 
-	ProgenyDTO getProgeny(final Integer gid);
+	ProgenyDTO getProgeny(Integer gid);
 
-	List<AttributeDTO> getAttributesByGUID
-		(
-			String germplasmUUID, List<String> attributeDbIds, Pageable pageable);
+	List<AttributeDTO> getAttributesByGUID(String germplasmUUID, List<String> attributeDbIds, Pageable pageable);
```

Comment:
These seem to be better suited in GermplasmAttributeService
```
- getAttributesByGID
- getAttributesByGUID
- countAttributesByGUID
- getGermplasmAttributeVariables
```

Reply:
done